### DISCUSSION TITLE
password while accessing firefox via web browser

### DISCUSSION THREAD

**Discussion by hyang1974** [jlesage/docker-firefox#201] [Q&A]
One question since I didn't find it after reading ReadMe...
I know password is enabled for VNC access, but does it support password protection via web browser? My use case is to access it via remote to reach to other PC/devices at my home using a browser, but it seems no password protection... any comments?

**Comment by hyang1974** [comment]
Closed... just realized that VNC password also apply to web access.

**Comment by troubleshootme** [comment]
Use Guacamole to access the container via vnc.
Just don't expose the container to the Internet, only guacamole should be exposed and will act as a gateway to the local instances. 
We use guacamole with sso setup to access the containers
We have several of these containers load-balanced with guacamole and it works well.
